examples/todoapp/lib/lib_permissions/models/permission_map_model.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:realm/realm.dart';
+part 'permission_map_model.realm.dart';
+
+@RealmModel(ObjectType.embeddedObject)
+class $PermissionMap {
+  late String key;
+  late bool value;
```

Comment:
Please document that `value` holds the information for `isEnabled` or change the name of the prop. In both ways the model properties should be documented

Reply:
Done